gnssanalysis/gn_io/sp3.py

```diff
@@ -829,6 +829,7 @@ def sp3_hlm_trans(a: _pd.DataFrame, b: _pd.DataFrame) -> tuple[_pd.DataFrame, li
     return b, hlm
 
 
+# Eugene: move to gn_diffaux.py (and other associated functions as well)?
```

Comment:
is this done?

Reply:
No, we need a discussion some time before moving the functions

Reply:
maybe label it as todo.

Reply:
Has been re-labelled to TODO - I would count as resolved